tests/Unit/Unit/AbstractUnitTest.php

```diff
@@ -25,7 +30,8 @@ class AbstractUnitTest extends TestCase
      */
     public function itShouldReturnCorrectName(string $name)
     {
-        $unit = new UnitStub($name, new ProcessBuilder());
+        $commandDispatcher = $this->prophesize(CommandDispatcherInterface::class);
+        $unit = new Service($name, $commandDispatcher->reveal());
```

Comment:
UnitStub is the Stub for the `AbstractUnit`.
It doing nothing is the point. It has no functionality and that is, what we want. We only want a testable instance that behaves like `AbstractUnit` but does not introduce any other functionality, as this is the only possible way to properly test abstract classes with Prophecy.